docs/faq.md

```diff
@@ -81,3 +81,14 @@ sudo systemctl disable cloud4rpi.service
 sudo rm /lib/systemd/system/cloud4rpi.service
 sudo systemctl daemon-reload
 ```
+
+
+## How to choose the DATA_SENDING_INTERVAL value?
+
+You should call [device.publish_data()](/api/python/#publish_data) and [device.publish_diag()](/api/python/#publish_diag) functions frequently to update the monitored variables. The recommended update frequency depends on the variable dynamics, for example, atmospheric conditions
+ do not change every minute. This means the delay between updates for a weather station should be one to ten minutes.
+
+If you monitor several variables with different dynamics, the delay between updates should be the shortest of all.
```

Comment:
should be the shortest of all??

Reply:
If one type of measurements performs every 5 minutes and the other is every minute, you should send data every minute.

Reply:
'the shortest' means the shortest.

Reply:
I don't see what's wrong. Short delay, shorter delay, the shortest delay. Is there some more useful word to specify small amount of time?